src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -66,8 +66,9 @@
               "The cloned Workspace will automatically inherit the Authorization Domain from this Workspace."
               [:div {} "You may add Groups to the Authorization Domain, but you may not remove existing ones."]])
            (this :-auth-domain-builder)
-           [comps/ErrorViewer {:error server-error}]
-           (style/create-validation-error-message validation-errors)])}]))
+           [:div {:style {:display "flex" :float "left"}}
```

Comment:
Is the `:float` doing anything here? I thought flex and float were different concepts and a quick search seems to agree, e.g.: https://stackoverflow.com/questions/39194630/float-does-not-work-in-a-flex-container/39194800#39194800